src/app/public/modules/date-range-picker/types/date-range-calculator.ts

```diff
@@ -60,6 +76,10 @@ export class SkyDateRangeCalculator {
     };
   }
 
+  /**
+   * Performs synchronous validation against the control.
+   * @param value
```

Comment:
Do we need a description for this param?

Reply:
Oops. Nope. Class methods don't even show up in the docs anyways. Removed the param.